### DISCUSSION TITLE
[Feature]Notification

### DISCUSSION THREAD

**Discussion by jinkazph** [1Panel-dev/1Panel#4443] [Ideas]
Hi. can we have a smtp for push notification. Or this features will be available on PRO? thanks.

**Comment by wanghe-fit2cloud** [comment]
Thank you for your feedback.  The notification and alerting features are planned to be supported in version 2.0 (expected to start researching related requirements in the second half of this year).

**Comment by wanghe-fit2cloud** [comment]
https://github.com/1Panel-dev/1Panel/issues/614
